app/assets/stylesheets/index.scss

```diff
@@ -24,30 +39,39 @@
 
 
 .piece-composer {
+  color: $black;
   font-size: 16px;
+  font-weight: bold;
   line-height: 1;
   margin: 0;
   padding: 0;
 }
 
 a {
+  color: $green;
   padding: 0;
 }
+
 .pagination {
+  font-size: 30px;
   text-align: center;
   width: 100%;
-  font-size: 30px;
 }
+
 .line {
-  align: center;
-  width: 50%;
+  display: block;
```

Comment:
Properties should be ordered border, border-top, display, height, margin-left, margin-top, width